common/data/orderUp.json

```diff
@@ -0,0 +1,5 @@
+[
+"1\t12 Nuggets -\t$4.39\n\tSauce: Ketchup\n\tLabel for:Alice",
+"2\tTots -\t$1.99\n\tLabel for:Bob",
+"1\t6 Wings -\t$6.29\n\tWing Seasoning: Sweet Honey BBQ\n\tLabel for:Charlie"
+]
```

Comment:
Can this just be plaintext? You can make requests that use text instead of JSON.